class AuthenticationError(Exception):
    pass


class PhoneNumbersError(Exception):
    pass
